### PR TITLE
nxp: imx8ulp: add audio-related DTS nodes

### PR DESCRIPTION
**Pull request by LaurentiuM1234**

### CODE REVIEW

**Review by iuliana-prodan — DISMISSED** [review]

**Review by mmahadevan108 — DISMISSED** [review]

**Comment by LaurentiuM1234** [comment]
> Please update the SHA in `west.yml`

done, thx

**Review by iuliana-prodan — DISMISSED** [review]

**Review by dbaluta — DISMISSED** [review]

**Review by mmahadevan108 — APPROVED** [review]

**Comment by mmahadevan108** [comment]
@dcpleung, could you please help review.

File: west.yml [hunk 1]
```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: f6c7108caf50684742689bfab838048208963f4f
+      revision: cd046faf8a3dc879982ff6079fc48a80a0d53e7c
```
Comment (mmahadevan108) [inline west.yml:196]: Please update SHA to 0afb4f2638812ae0c1c56973acb7f9278741da85

**Review by mmahadevan108 — CHANGES_REQUESTED** [review]
Please update SHA to only pull in your changes.

**Comment by LaurentiuM1234** [comment]
> Please update SHA to only pull in your changes.

done

**Review by iuliana-prodan — DISMISSED** [review]

**Review by iuliana-prodan — APPROVED** [review]

**Review by mmahadevan108 — APPROVED** [review]

**Comment by mmahadevan108** [comment]
@dcpleung, this PR is ready for your review.

**Review by dcpleung — APPROVED** [review]

**Pull request merged**